server_environment/__manifest__.py

```diff
@@ -31,6 +31,4 @@
     "data": [
         'serv_config.xml',
     ],
-    'installable': False,
```

Comment:
This key is actually required for the scripts that are used to disable all the addons in a new branch when a new Odoo version is released. From what I understand, things break hard without it.